app/models/apple/podcast_delivery_file.rb

```diff
@@ -327,14 +327,14 @@ def delivered?
     def processed_errors?
       return false unless asset_processing_state.present?
 
-      processed_validation_failed? || processed_duplicate?
```

Comment:
Uploading a duplicate file is not really an error.

Reply:
+1